pkg/crc/services/dns/dns.go

```diff
@@ -55,7 +55,11 @@ func RunPostStart(serviceConfig services.ServicePostStartConfig) (services.Servi
 		return *result, err
 	}
 
-	orgResolvValues := network.GetResolvValuesFromInstance(serviceConfig.Driver)
+	// We need to restart the Host Network before updating
+	// the VM's /etc/resolv.conf file.
+	res, err := runPostStartForOS(serviceConfig, result)
```

Comment:
what if an `err` occurs here?

Reply:
We are returning it https://github.com/code-ready/crc/pull/234/files#diff-77b86e271b726efdfa282a22b5c5fd27R75 here.